csharp/ql/lib/semmle/code/csharp/frameworks/system/Web.qll

```diff
@@ -3,6 +3,7 @@
 import csharp
 private import semmle.code.csharp.frameworks.System
 private import semmle.code.csharp.frameworks.system.collections.Specialized
+private import semmle.code.csharp.dataflow.ExternalFlow
```

Comment:
Add a `private import semmle.code.csharp.frameworks.system.Web` to the module `Frameworks` in `ExternalFlow.qll` to ensure bidirectional importing.